pymc/gp/cov.py

```diff
@@ -518,6 +521,7 @@ def __init__(
         ls=None,
         ls_inv=None,
         active_dims: Optional[IntSequence] = None,
+        square_dist: Optional[Callable] = None,
```

Comment:
I can't see where the `square_dist` argument is being used later. Do you want to attach it to the covariance instance at the end of `__init__`? As it stands now, it looks like this isn't doing anything. Furthermore, even if you attach it to the object, would that be called instead of the `__class__` `square_dist` accessor?